<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>268</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 170, 0);
font: italic 16pt &quot;Harlow Solid Italic&quot;;</string>
            </property>
            <property name="text">
             <string>1ra Ley de Newton (Ley de la Inercia)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_cerrar">
            <property name="text">
             <string>Cerrar</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>268</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="styleSheet">
             <string notr="true">QPushButton{
border-color: rgb(170, 0, 0);
font: 81 italic 11pt &quot;Montserrat ExtraBold&quot;;
}

QLabel{
font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QDoubleSpinBox{
font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QCommandLinkButton{
color: rgb(0, 170, 255);
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,1,1,1,8">
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="plainText">
                <string>Primera ley de Newton: un objeto en reposo permanece en reposo o, si está en movimiento, permanece en movimiento a una velocidad constante, a menos que una fuerza externa neta actúe sobre él.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Para más información puedes checar el siguiente link:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCommandLinkButton" name="linkButton_pag">
               <property name="text">
                <string>¿Qué es la primera ley de Newton?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_comenzar">
               <property name="text">
                <string>Comenzar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1,1,1,0,1">
                <item>
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="2" column="2">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Voz :</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Voy :</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QDoubleSpinBox" name="sBox_Vox">
                    <property name="minimum">
                     <double>-1000.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1000.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QDoubleSpinBox" name="sBox_Voy">
                    <property name="minimum">
                     <double>-1000.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1000.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Vox :</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="2">
                   <widget class="QDoubleSpinBox" name="sBox_Voz">
                    <property name="minimum">
                     <double>-1000.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1000.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Xo :</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Yo :</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Zo :</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QDoubleSpinBox" name="sBox_Xo">
                    <property name="minimum">
                     <double>-1000.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1000.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QDoubleSpinBox" name="sBox_Yo">
                    <property name="minimum">
                     <double>-1000.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1000.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QDoubleSpinBox" name="sBox_Zo">
                    <property name="minimum">
                     <double>-1000.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1000.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Tiempo :</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="sBox_tiempo">
                  <property name="minimum">
                   <double>-1000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_graficar">
                  <property name="text">
                   <string>Graficar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_guardar">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Guardar Gráfica (GIF)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_reiniciar">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Reiniciar</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,5,8">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Datos Obtenidos:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="tabla_valores">
               <column>
                <property name="text">
                 <string>Tiempo [s]</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Posición X [m]</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Posición Y [m]</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Posición Z [m]</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Velocidad X [m/s]</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Velocidad Y [m/s]</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Velocidad Z [m/s]</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="graf_1ra_ley"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
